source/qml_v2/AdaptiveCardQmlEngine/qml/TextInputRender.qml

```diff
@@ -118,7 +118,6 @@ Column {
     
     InputErrorMessage {
         id: inputtextErrorMessage
-        
         visible: showErrorMessage
```

Comment:
Why this change ?

Reply:
There was no changes in this file. this change happened by mistake.